src/Docfx.App/RunServe.cs

```diff
@@ -41,7 +41,7 @@ public static void Exec(string folder, string host, int? port, bool openBrowser,
             builder.Logging.ClearProviders();
             builder.WebHost.UseUrls(url);
 
-            Console.WriteLine($"Serving \"{folder}\" on {url}. Press Ctrl+C to shut down.");
+            Console.WriteLine($"Serving \"{folder}\" on {url}{Environment.NewLine}Press Ctrl+C to shut down.");
```

Comment:
```suggestion
            Console.WriteLine($"Serving \"{folder}\" on {url}");
            Console.WriteLine("Press Ctrl+C to shut down");
```

(important part being to drop the last `.`, using two line for readability of the intent but this is just my opinion and is not important)